### PR TITLE
Registration final page

### LINKED ISSUE #42: Registration Final Page: Create submit with different kinds of membership options
- have a terms and conditions
- have membership options
- integrate with stripe

### PR DESCRIPTION
**Pull request by mohaimenhasan**
Closes #42 

Created the final page. I only need to integrate Stripe with this. Putting Stripe in another task

### CODE REVIEW

File: src/Components/Registration/FinalPage.js [hunk 1]
```diff
@@ -1,14 +1,555 @@
...
         );
     }
 }
 
-export default FinalPage;
\ No newline at end of file
+FinalPage = withMyHook(FinalPage);
+export default FinalPage;
+
+/*
```
Comment (ahmedhamodi) [inline src/Components/Registration/FinalPage.js:496]: Is there a reason you are keeping all this uncommented code?

File: src/Components/Registration/FinalPage.js [hunk 2]
```diff
@@ -1,14 +1,555 @@
...
+                            <Card className={classes.cardRoot}>
+                                <CardActionArea>
+                                    <CardMedia
+                                        className={classes.media}
+                                        image={AspiringMember}
+                                        title="Contemplative Reptile"
+                                    />
+                                    <CardContent>
+                                        <Typography gutterBottom variant="h5" component="h2">
+                                            <b> Mentee </b>
```
Comment (ahmedhamodi) [inline src/Components/Registration/FinalPage.js:253]: We should use actual body text here. On the landing page we have the titles and body text that you can use. If anything, we should just reload that component so we don't need to modify in multiple places.

File: src/Components/Registration/FinalPage.js (hunk 1 above)
Comment (mohaimenhasan) [reply src/Components/Registration/FinalPage.js:496]: my bad I forgot to take it out

File: src/Components/Registration/FinalPage.js (hunk 2 above)
Comment (mohaimenhasan) [reply src/Components/Registration/FinalPage.js:253]: The membership texts are different right? 
**This is the main page: **
image


**These are the two membership options: **
image

Comment (ahmedhamodi) [reply src/Components/Registration/FinalPage.js:253]: Hmm I see. So we'll update this later with verbiage from Sammer or Aazar.

File: src/Components/Authentication/SignIn.js [hunk 3]
```diff
@@ -32,11 +34,12 @@ const useStyles = makeStyles((theme) => ({
     },
     // this css element describes the size of the image
     img: {
+        cursor: 'pointer',
         width: '150px',
         padding: '1vw',
     },
     image: {
-        backgroundImage: 'url(https://i.picsum.photos/id/1003/1181/1772.jpg)',
+        backgroundImage: `url(${signInImage})`, //'url(https://i.picsum.photos/id/1003/1181/1772.jpg)',
```
Comment (ahmedhamodi) [inline src/Components/Authentication/SignIn.js:42]: URL is not working, so using static image for now.

**Pull request merged**